docs/learn/response_models.md

```diff
@@ -250,6 +250,37 @@ Once exhausted, you can access the final, full response model through the `const
 
 You can also use the `stream` property to access the `BaseStream` instance and [all of it's properties](./streams.md#common-stream-properties-and-methods).
 
+## FromCallArgs
+
+`FromCallArgs` is a feature in Mirascope that allows you to directly incorporate function arguments into your response models, enabling seamless integration between LLM outputs and function inputs.
```

Comment:
nit: combine this with the following sentence:

"Fields annotated with `FromCallArgs` will be populated with the corresponding argument from the function call rather than expecting it from the LLM's response. This enable seamless validation of LLM outputs against function inputs:"